backend/docs/api/inventory/management_v1.yaml

```diff
@@ -674,7 +674,6 @@ components:
           type: string
           description: Attribute description.
         value:
-          type: string
```

Comment:
I am having some doubts, have you tested it? what would the oapi-codegen generate? why not use `oneOf`?

Reply:
The model code says https://github.com/mzedel/mender-server/blob/main/backend/services/inventory/model/device.go#L67 - so I thought to be correct that would be the way, OTOH `oneOf` would be more helpful => I'll adjust!

As to oapi-codegen: we can have this as a requirement as soon as we have anything actually relying on this - so far the only generators actually using the specs seem to be outside of the backend.